book-a-reading-room-visit.test/EmailTemplate/Text/ReservationConfirmation.txt

```diff
@@ -13,7 +13,6 @@
       Visit type: {VisitType}
       Visit date:   {VisitStartDate}
       Reading room: {ReadingRoom}
-      Seat number:  {SeatNumber}
               
       Based on availability, we may need to change your seat on the date of your visit.
```

Comment:
I wonder should we remove "Based on availability, we may need to change your seat on the date of your visit." too.